package.json

```diff
@@ -111,13 +111,13 @@
     "chokidar-cli": "^3.0.0",
     "eslint": "^8.57.0",
     "eslint-plugin-jsx-a11y": "^6.8.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react": "^7.34.1",
     "optipng-bin": "^9.0.0",
-    "prettier": "^2.8.8",
+    "prettier": "^3.2.5",
     "stylelint": "^15.10.2",
     "stylelint-config-standard": "^34.0.0",
-    "stylelint-prettier": "^3.0.0",
+    "stylelint-prettier": "^4.1.0",
```

Comment:
Updating `stylelint-prettier` for the same reason. See https://github.com/prettier/stylelint-prettier/blob/main/CHANGELOG.md#400-2023-07-07